deployment/network-operator/templates/sriovnetwork.openshift.io_sriovoperatorconfig.yaml

```diff
@@ -21,6 +21,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   # Add fields here
+  {{- with (index .Values "sriov-network-operator" "operator") }}
+  enableInjector: {{ .enableAdmissionController }}
+  enableOperatorWebhook: {{ .enableAdmissionController }}
+  {{- end }}
   enableInjector: false
   enableOperatorWebhook: false
```

Comment:
Should we remove the `enableOperatorWebhook: false` on line 29?

Reply:
yes!